### ISSUE TITLE
Improve ARIA roles for accessibility

### ISSUE THREAD

**Issue by lacymorrow** [Simonwep/pickr#167]
Testing accessibility with Google's [Accessibility Developer Tools](https://github.com/GoogleChrome/accessibility-developer-tools) and there are a few ARIA roles which should be updated. _Electron_

```
Accessibilty audit failed

Error: AX_ARIA_01 (Elements with ARIA roles must use a valid, non-abstract ARIA role) failed on the following elements (1 - 4 of 4):
body > .pcr-app
body > .pcr-app > .pcr-selection > .pcr-color-palette > .pcr-palette
body > .pcr-app > .pcr-selection > .pcr-color-chooser > .pcr-hue.pcr-slider
body > .pcr-app > .pcr-selection > .pcr-color-opacity > .pcr-opacity.pcr-slider
See https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules#ax_aria_01 for more information.
```

I use `app.client.auditAccessibility()` via Spectron but the output is the same on the web

#### Suggested

> My opinions, there may be more valid non-abstract roles listed here: https://www.w3.org/TR/wai-aria-1.0/roles#roles_categorization

 - `.pcr-app` from `role="window"` to `role="form"`
 - `.pcr-palette` from `role="widget"` to `role="listbox"`
 - `.pcr-opacity` and `.pcr-hue` from `role="widget"` to `role="slider"`

#### My environment:
```
Version (see Pickr.version): 1.4.2
Used bundle (es5 or normal one): normal
Used theme (default is classic): nano
Electron 6.0.11
darwin 18.7.0
```

**Comment by Simonwep** [comment]
That's really nice, didn't knew there where such specific roles available :D
